### DISCUSSION TITLE
RustDesk Fork with custom servers and key not working

### DISCUSSION THREAD

**Discussion by mdtabrezpasha** [rustdesk/rustdesk#7237] [Q&A]
Hi,
    I created a fork of rustdesk and added my server's address and key to it, but the setup files being generated do not have the setting baked in.  is this guide wrong?. https://rustdesk.com/docs/en/dev/build/all/

**Comment by TroySchulz** [comment]
I am having the exact same issue following instructions on the same page with the same result (or lack thereof).

**Comment by HDTS1** [comment]
I am having the same problem still.

**Comment by NoirPi** [comment]
Yeah same problem here.
